src/utilities/copyTextToClipboard.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copies a piece of text to the clipboard. Adapted from [StackOverflow][1].
+ *
+ * [1]: https://stackoverflow.com/a/30810322
```

Comment:
I'm not sure why we had this reference, but let's just remove it
```suggestion
 * Copies a piece of text to the clipboard.
 *
```

Reply:
Done.